### PR TITLE
Adding integration of the update modals in the showModules page

### PR DESCRIPTION
**Pull request by Hai5edfm**
This PR adds the update modals, still needs the visual feedback and the movies final integration

### CODE REVIEW

File: src/hooks/useGetData.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+              }
+            }
+          )
+          .then((response) => {
+            if(response.data.data?.showAllUsers) setModuleData(response.data.data.showAllUsers)
+            if(response.data.data?.showAllMovies) setModuleData(response.data.data.showAllMovies)
+            if(response.data.data?.showAllWriters) setModuleData(response.data.data.showAllWriters)
+            if(response.data.data?.showAllDirectors) setModuleData(response.data.data.showAllDirectors)
+            if(response.data.data?.showAllMusicians) setModuleData(response.data.data.showAllMusicians)
+            console.log('response: ', response.data.data.showAllUsers);
```
Comment (LilianaRestrepoTorres) [inline src/hooks/useGetData.js:26]: We should delete every console.log in a next PR.

File: src/hooks/useGetData.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
```
Comment (LilianaRestrepoTorres) [inline src/hooks/useGetData.js:1]: We don't need this hook.

File: src/pages/showmodules/index.tsx [hunk 3]
```diff
@@ -14,43 +14,39 @@ function ShowModules() {
...
   } else {
     orderBy = "Ascendent";
   }
 
   // Calling the hook which contains the state and return the data of items
-  const { items, searchValue, orderItems, filterStatus } =
-    useGetModules("public2");
-
+  const {items, searchValue, orderItems, filterStatus} = useGetModules(titleModule);
+  console.log(items)
```
Comment (LilianaRestrepoTorres) [inline src/pages/showmodules/index.tsx:32]: It should be deleted.

**Review by LilianaRestrepoTorres — APPROVED** [review]
We can delete 'src/schemas/updateItem.ts' file it's already empty.

Good job! @Hai5edfm

**Review by AreYouIvan — APPROVED** [review]

**Review by AreYouIvan — CHANGES_REQUESTED** [review]
Check the errors from production
image

**Pull request merged**